WikiWatershed/model-my-watershed
# Revert explicit save

## Overview
After our attempt at #1995 we realized warning the user when they try to leave unsaved work is a lot more involved than one would hope. We're reverting the explicit-save feature for this release. 

Connects #1997

## Testing Instructions

 * Pull, and `bundle`
 * While not logged, model an AoI, then log in — confirm a project is created and saved
 * While logged in confirm every time you make it to model a project is created/saved and any subsequent changes to it are also automatically saved
